corehq/apps/users/bulkupload.py

```diff
@@ -487,12 +487,14 @@ def is_password(password):
                     if language:
                         user.language = language
                     if email:
-                        email_validator = EmailValidator()
                         try:
-                            email_validator(email)
-                            user.email = email
+                            validate_email(email)
                         except ValidationError:
                             raise UserUploadError(_("User has an invalid email address"))
+
+                        if not email.islower():
+                            raise UserUploadError(_("Email address must be lower case: {}".format(email)))
```

Comment:
http://stackoverflow.com/questions/9807909/are-email-addresses-case-sensitive

take a look at that answer and the comments. not the most in depth research, so maybe i'll dive in a bit more to see if i can get something more definitive